person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.ran(1..1000)
```

Comment:
- [ ] You have a little mistake here the method that you are trying to use with `Random` is `.rand(...)` and not `.ran(...)`, please change this 😄

Reply:
Thank you